### PR TITLE
Escape html in comments in author info from DESCRIPTION

### LINKED ISSUE #1205: html in authors info from DESCRIPTION is not escaped
Example 

* source https://github.com/ropensci/refsplitr/blob/master/DESCRIPTION#L25

* authors page https://docs.ropensci.org/refsplitr/authors.html

### PR DESCRIPTION
**Pull request by maelle**
cc @jeroen 

fix #1205

### CODE REVIEW

**Comment by maelle** [comment]
Maybe it should even use `linkify()`? :thinking: 

Example

``` r
comment <- "Kramer reviewed the package for ropensci,
             see <https://github.com/ropensci/software-review/issues/256>"

pkgdown:::linkify(comment)
#> [1] "Kramer reviewed the package for ropensci,\n             see &lt;<a href='https://github.com/ropensci/software-review/issues/256'>https://github.com/ropensci/software-review/issues/256</a>&gt;"
```

<sup>Created on 2020-01-16 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by jeroen** [comment]
Thanks, looks good!

File: tests/testthat/test-build-home-authors.R [hunk 1]
```diff
@@ -26,3 +26,14 @@ test_that("names can be removed from persons", {
   expect_length(remove_name(p4$comment, "ORCID"), 0)
   expect_length(remove_name(p5$comment, "ORCID"), 0)
 })
+
+test_that("Comments in authors info are linkified", {
+  p <- list(name = "Jane Doe",
```
Comment (hadley) [inline tests/testthat/test-build-home-authors.R]: Can you please use standard tidyverse styling?

Comment (maelle) [reply tests/testthat/test-build-home-authors.R]: Oops, now I think it looks like https://style.tidyverse.org/syntax.html#long-lines :-)

**Comment by hadley** [comment]
Thanks!

**Pull request merged**